### ISSUE TITLE
2 blank lines added at the end of file, conflicts with markdownlint

### ISSUE THREAD

**Issue by rowild** [sumnow/markdown-formatter#24]
Hi!

The extension adds 2 new lines at the end of the file, which conflicts with linting rules. Is there a setting to suppress that?

I added these rules, but they do not make a difference:

```
   "markdownFormatter.formatOpt": {
      "end-with-newline": false,
      "indent_size": 2,
      "preserve-newlines": false,
      "max-preserve-newlines": 1
    },
```

(When I disable the markdown-formatter extension, no empty lines are added, so I assume it is something that this extension does and not VScode...)

Thank you!

**Comment by sumnow** [comment]
I'm sorry about this issue.
It has been fixed in the [0.8.1](https://github.com/sumnow/markdown-formatter/blob/master/CHANGELOG.md#081) version. Thank you.

**Comment by rowild** [comment]
Thank you very much!!!
